js/views/message.js

```diff
@@ -152,7 +217,6 @@ views.NoSearchResultMessageListView = Marionette.ItemView.extend({
 	template: "#no-search-results-message-list-template",
 
 	onRender: function() {
-		$('#load-more-mail-messages').hide();
 	}
```

Comment:
That is not the latest. Go to the commits tab of this pull, and you can see the latest state/commit.
https://github.com/owncloud/mail/commit/d276088f99401bb14043cdd2ee09e85170fbda95